### DISCUSSION TITLE
カスタマイズメニュー（ファイル名、タグタイトル）2022年度講座更新

### DISCUSSION THREAD

**Discussion by CSReviser** [CSReviser/CaptureStream#12] [General]
旧アプリではあった機能の一部を試験的に設置しています。
更新できていなかったカスタマイズメニュー（ファイル名、タグタイトル）を2022年度講座に合わせて修正更新しました。
使えなくなっていた欧州系の「まいにちXXX語講座」については【月火水】と【木金】の講座を共通の設定となっています。
バグが残っている可能性があり、正常に機能するか、また、今後継続メンテ可能かは保証できません。

※ffmpegはバージョン5.0.1に対応しています。
※Mac版は環境がないので、動作未確認です。

<hr /><em>This discussion was created from the release <a href='https://github.com/CSReviser/CaptureStream/releases/tag/20220523'>カスタマイズメニュー（ファイル名、タグタイトル）2022年度講座更新</a>.</em>

**Comment by DaddyJapan** [comment]
CaptureStream-Macintosh-20220523.dmgのいろんなテストはしていませんが、基本的には動作確認しました。また、CS-Multiも動作確認をしております。CS-Multiの方には、Discussionsがないので、こちらに記載します。いつもありがとうございます。

**Reply by CSReviser** [reply]
動作確認ありがとうございます。更新できずに放置していたカスタマイズメニューを更新など、録音に直接直接影響しない更新が続くかもしれません。使わない機能の更新はスキップしていただいて良いと思います。
